Probesys/bileto
# new: Create mailboxes

## Related issue(s)



https://github.com/Probesys/bileto/issues/331

## Changes



- improve the `.row` CSS classes
- add PHP-IMAP as a dependency
- create an Encryptor class
- create a Mailbox entity + migration + factory
- add a `admin:manage:mailboxes` permission
- add an `inbox` icon
- create a controller to list the mailboxes
- add a form to create mailboxes
- add a button to test the connection to mailboxes

## How to test manually



- create a mailbox as explained in the GreenMail documentation
- check the connection works

## Checklist
